lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -283,11 +285,11 @@ private function _get_imagetools_configuration()
         }
 
         return <<<EOT
-imagetools_toolbar: "editimage imageoptions",
-images_upload_url: "{$url}",
-images_upload_base_path: "/",
-images_upload_credentials: true,
+imagetools_toolbar: "editimage imageoptions",S
 imagetools_cors_hosts: ['{$hostname}'],
+original: "",
```

Comment:
we've already had this version a couple of revisions ago, is there really no better way to handle this?